src/azure-cli/azure/cli/command_modules/sql/_help.py

```diff
@@ -457,6 +457,11 @@
 short-summary: Updates the instance failover group.
 """
 
+helps['sql instance-pool'] = """
+type: group
+short-summary: Manage instance pools.
+"""
+
```

Comment:
can we also add help entry for all your new commands with short-summary and some examples?

Reply:
ACK'd

Reply:
> Hey @Juliehzl do you mean something like the comments in "sql db copy"?

I mean for each new command in `az sql instance-pool` command group. e.g. for `az sql instance-pool create/delete/list/show/update`